inc/Engine/Optimization/DelayJS/HTML.php

```diff
@@ -101,14 +101,25 @@ public function delay_js( $html ): string {
 		$this->excluded = array_merge( $this->excluded, $this->options->get( 'delay_js_exclusions', [] ) );
 		$this->excluded = array_merge( $this->excluded, $this->options->get( 'delay_js_exclusions_selected_exclusions', [] ) );
 
+		if ( $this->options->get( 'delay_js_execution_safe_mode', 0 ) ) {
+			$this->excluded = array_merge(
+				$this->excluded,
+				[
+					'/jquery(-migrate)?-?([0-9.]+)?(.min|.slim|.slim.min)?.js(\?(.*))?( |\'|"|>)',
+					'js-(before|after)',
+					'(?:/wp-content/|/wp-includes/)(.*)',
+				]
```

Comment:
Is there a reason you're using the hardcoded values instead of the static method here?

Reply:
Umm nope, I forgot to change it, I've used the hardcoded for some tests during the tests fixing stage. Forgot to revert back to the static method. Thanks !